diff --git a/soft-hwclock b/soft-hwclock
index 70fe386..bcf573c 100644
--- a/soft-hwclock
+++ b/soft-hwclock
@@ -1,10 +1,10 @@
 #!/bin/sh

-# Get the saved time (or empty) and clock time, in RFC399 format
+# Get the saved time (or empty) and clock time, in ISO format
 # This format sorts lexicographically

 CLOCKFILE="/opt/soft-hwclock/soft-hwclock.data"
-CTIME=$(date --rfc-3339=seconds)
+CTIME=$(date "+%C%y-%m-%d %H:%M:%S-00:00")
 FTIME=$(test -f "${CLOCKFILE}" && cat "${CLOCKFILE}")

 # if the file time is in the future, use that